tool-plugins/nextjs-starter/src/components/Example.tsx

```diff
@@ -0,0 +1,27 @@
+import { APP_BRIDGE_TOKEN_HEADER_KEY, KEY_TOKEN } from '@/utils/const';
+import { useEffect, useState } from 'react';
+
+export default function Test() {
```

Comment:
Hey @eunjae-lee 🙌 

Should the places using `Test` be renamed to `Example` for better clarity?

Reply:
done!